### DISCUSSION TITLE
Data analysis on Datasets leaked by Anonymous

### DISCUSSION THREAD

**Discussion by mossishahi** [WeAreMahsaAmini/Core#254] [Ideas]
Anonymous group has leaked some datasets, maybe we should collect them in a repo here and extract some information from them.

**Comment by ZahraEconomist** [comment]
That sounds great, where can we find those datasets?

**Reply by mossishahi** [reply]
> That sounds great, where can we find those datasets?

Some datasets I found in a telegram channel sounds to be for Anonymous group:
- [Ministry of Cooperation, Labor and Social Welfare HACKED!](https://t.me/YourAnonSpid3r/296)
- [yazd.iau.ir | Islamic Azad University HACKED! Database breached. ](https://t.me/YourAnonSpid3r/317)
- [Forensic Research Center of Iran HACKED! 100 Mb SQL Database Breached](https://mega.nz/file/EQ1UxaYK#fpcH6kmZvdpDBw1pmJUMlMoCVznMF0HYYibdB9cbJD8)
- [Fars News Agency Hacked! Here is 30K record from Database.](https://sendspace.com/file/1gqump)

**Reply by neginbandar** [reply]
Hey, Are these files safe? What kind of data is available?

**Reply by mossishahi** [reply]
> Hey, Are these files safe? What kind of data is available?

As I mentioned a direct link to their telegram channel, I adopted them from there.
